content/learn/accessibility/code-review/index.md

```diff
@@ -28,7 +28,10 @@ Ensure that link text describes the target and purpose and that it can be distin
 
 ## Semantic markup
 
-Ensure that proper semantic HTML elements are used to describe the content of the page or component. For example, <code>&lt;ul&gt;</code> unordered lists are used for lists, <code>&lt;h1&gt;</code>-<code>&lt;h6&gt;</code> are used in order for headings, <code>&lt;nav&gt;</code> is used for navigation.
+Ensure that proper semantic HTML elements are used to describe the content of the page or component. For example:
+- <code>&lt;ul&gt;</code> are used for unordered lists
+- <code>&lt;h1&gt;</code>-<code>&lt;h6&gt;</code> are used in order for headings
```

Comment:
It might read better to change the hyphen to "through" between the code snippets. ... Also, to make this use active voice, I'd start with "Use" and delete "are used."  If we do this, the bullet becomes a complete sentence, so we should also end with a period. (i.e., "Use &lt;h1&gt; through &lt;h6&gt; in order for headings.")